@navikt/navno/react/src/card/base-card/types.ts

```diff
@@ -0,0 +1,41 @@
+export enum CardType {
+  Product = "product",
+  Situation = "situation",
+  Tool = "tool",
+}
+
+export type CardSize = "large" | "mini" | "micro";
+
+export enum Interaction {
+  mouseenter = "mouseenter",
+  mouseleave = "mouseleave",
+  mousedown = "mousedown",
+  mouseup = "mouseup",
+  touchend = "touchend",
+  touchstart = "touchstart",
+  touchmove = "touchmove",
+  touchcancel = "touchcancel",
+  focus = "focus",
+  blur = "blur",
+}
+
+export type ProductName = "barnepensjon" | "engangsstonad";
+
+export type Language = "no" | "en";
```

Comment:
@bdahle og jeg har snakket om å implementere en løsning for i18n i designsystemet som ligner på den de har i Adobe Spectrum React (https://react-spectrum.adobe.com/react-aria/internationalization.html). Hva tenker dere om en sånn løsning, @terjeofnorway og @KenAJoh?

Reply:
💯 En slik løsning har vært supert å få inn i systemet!